packages/i18n/tests/ui/specs/Loader/Loader-specs.js

```diff
@@ -1,89 +1,89 @@
+/* eslint-disable jest/valid-expect */
```

Comment:
🤔 I don't see any warnings related to this from my console.

Reply:
I removed this disable also.